components/kyma-environment-broker/internal/storage/tests_utils.go

```diff
@@ -291,43 +188,44 @@ func SetupTestDBTables(connectionURL string) (cleanupFunc func(), err error) {
 }
 
 func isDockerTestNetworkPresent(ctx context.Context) (bool, error) {
-	netReq := testcontainers.NetworkRequest{
-		Name:   DockerUserNetwork,
-		Driver: "bridge",
-	}
-	provider, err := testcontainers.NewDockerProvider()
 
-	if err != nil || provider == nil {
-		return false, errors.Wrap(err, "Failed to use Docker provider to access network information")
+	cli, err := client.NewClientWithOpts(client.FromEnv)
+	if err != nil {
+		panic(err)
 	}
 
-	_, err = provider.GetNetwork(ctx, netReq)
+	filters := filters.NewArgs()
```

Comment:
nit: I personally try to avoid shadowing a package name by variable name, consider please either changing the import name or variable name to avoid shadowing.